content/blog/incubation.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Incubation level project in CNCF"
+description = "CoreDNS is now an incubation level project in the CNCF."
+tags = ["CNCF", "project"]
+date = "2018-02-26T13:10:29+08:00"
+author = "coredns"
++++
+
+CoreDNS has just graduated to the
+[incubation](https://twitter.com/CloudNativeFdn/status/968223259643858951) level in the
+[CNCF](https://cncf.io). This means we're now on the same footing as, say, the Kubernetes project.
```

Comment:
K8s just graduated out of incubation. Prometheus and Fluentd will soon too. How about gRPC and Envoy?